docs/source/_templates/page.html

```diff
@@ -16,7 +16,7 @@
       document.getElementById("doc_ver_warning").innerHTML = `
       <div class="admonition warning">
       <p class="admonition-title">Warning</p>
-      <p>This version of the documentation corresponds to the master branch of <code class="docutils literal notranslate"><span class="pre">cleanlab</span></code> source code from <a href="https://github.com/cleanlab/cleanlab/">GitHub</a>. To see the documentation for the latest <code class="docutils literal notranslate"><span class="pre">pip</span></code>-installed version, click <a href="../stable/index.html">here</a>.</p>
+      <p>This version of the documentation corresponds to the master branch of <code class="docutils literal notranslate"><span class="pre">cleanlab</span></code> source code from <a href="https://github.com/cleanlab/cleanlab/">GitHub</a>. To see the documentation for the latest <code class="docutils literal notranslate"><span class="pre">pip</span></code>-installed version, click <a href="/cleanlab-docs/stable/index.html">here</a>.</p>
```

Comment:
Not sure how your mockup site is setup exactly, but when I click on `master` (under Versions), the banner that appears in the linked docs (https://weijinglok.github.io/cleanlab-docs/add-img-tut/index.html) wrongly says this: 

`Warning: This documentation is for an old version (add-img-tut) of cleanlab. To see the documentation for the latest stable version (v3.0.1), click [here](https://weijinglok.github.io/cleanlab-docs/stable/index.html).`

The banner should instead say the above.

Reply:
Oh, that's because i've not used the `master` branch in my mockup, I've used the `add-img-tut` branch instead. The logic goes "if it's not on the latest, stable branch, and not on a branch called `master`, it must be an old branch" so it displays that message instead. but this error won't happen when we deploy to production (cleanlab\cleanlab's `master` branch).